### ISSUE TITLE
well known proto not building

### ISSUE THREAD

**Issue by makdharma** [grpc/grpc#9730]
Repro instructions.

```
$ cat BUILD
proto_library (
    name = "test_proto",
    srcs = ["test.proto"]
)

$ cat test.proto
syntax = "proto3";
package test;
import "google/protobuf/any.proto";
message Test {
  google.protobuf.Any anything = 1;
}

$ bazel build :test_proto
INFO: Found 1 target...
ERROR: /usr/local/google/home/makarandd/grpc/hack/BUILD:1:1: Generating Descriptor Set proto_library //hack:test_proto failed: Process exited with status 1 [sandboxed].
google/protobuf/any.proto: File not found.
hack/test.proto: Import "google/protobuf/any.proto" was not found or had errors.
hack/test.proto:5:3: "google.protobuf.Any" is not defined.
Use --strategy=GenProto=standalone to disable sandboxing for the failing actions.
Target //hack:test_proto failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 0.397s, Critical Path: 0.10s
```

**Comment by nicolasnoble** [comment]
Didn't you fix it, @makdharma ?

**Comment by ncteisen** [comment]
I fixed this for #13883
